### ISSUE TITLE
[Broken Website] Westlaw.com blue highlighting is broken and showing up as grey (instead of blue)

### ISSUE THREAD

**Issue by faroutnovelty** [darkreader/darkreader#9688]
# Broken Website Report

## Website Address
https://1.next.westlaw.com

([And here is the page in the screenshots](https://1.next.westlaw.com/Document/I5c0d7c889cbd11d9bdd1cfdd544ca3a4/View/FullText.html?transitionType=UniqueDocItem&contextData=(sc.Default)&userEnteredCitation=268+U.S.+203))

## Steps To Reproduce
Here is an example:
- Go to https://1.next.westlaw.com/Search/Home.html?transitionType=Default&contextData=(sc.Default)&firstPage=true&bhcp=1
- Go to any document, case, or text
- Highlight the text
- Click the blue highlight option provided by Westlaw.com

## Problematic Observation
When Dark Reader is enabled, blue highlights show up as grey (the same color as grey highlights). No other Westlaw highlight colors are affected besides blue highlighting.

## Screenshots
**This is where to select blue highlighting:**
<img width="999" alt="Screen Shot 2022-08-28 at 8 24 47 AM" src="https://user-images.githubusercontent.com/88589076/187074170-1f0cf6d2-c8bf-4abd-a30e-59fd9ff0fdec.png">

**The is what blue highlighting looks like with Dark Reader turned on:**
<img width="998" alt="Screen Shot 2022-08-28 at 8 24 56 AM" src="https://user-images.githubusercontent.com/88589076/187074178-695103c8-c995-4b59-9778-4759bfd5f33f.png">

**And this is what blue highlights look like when Dark Reader is turned off:**
<img width="1010" alt="Screen Shot 2022-08-28 at 8 25 07 AM" src="https://user-images.githubusercontent.com/88589076/187074190-2f72d0e9-6070-41ab-bc25-1a3ab6886fa6.png">

**And here's a screenshot of the developer tools with the blue highlight element selected**
<img width="2050" alt="Screen Shot 2022-08-30 at 8 01 09 PM" src="https://user-images.githubusercontent.com/88589076/187564957-a9c9408d-f370-45c8-8464-29c6e8d811d5.png">

## System Information
OS: MacOS 12.5.1

Browser: Chrome 104.0.5112.101
Dark Reader: Dark Reader Version 5 Preview (4.9.57)

&

Browser: Safari 15.6.1
Dark Reader: Dark Reader 1.4.6.

## Additional Context
This issue has persisted for over a year now and is particularly annoying because each discreet color of highlighting is very important.

**Comment by faroutnovelty** [comment]
I'll donate $150 to Dark Reader via Open Collective as soon as the issue gets fixed.

**Comment by faroutnovelty** [comment]
Here's a screenshot of the developer tools in case you are not able to log in.
<img width="2050" alt="Screen Shot 2022-08-30 at 8 01 09 PM" src="https://user-images.githubusercontent.com/88589076/187564841-a34c3e16-cca9-44bf-8a9f-09c891229737.png">

**Comment by Gusted** [comment]
Hi @faroutnovelty! Thank you for your detailed screenshots with the devtools included. Unfortunately the screenshots doesn't show which rule(with their respective selector + styles) is being applied that does the highlighting. Could you make a new screenshot (with dark reader enabled) with the same element selected and being highlighted by the web instead of by the browser as currently shown in the screenshots (those with devtools included).

**Comment by faroutnovelty** [comment]
> Hi @faroutnovelty! Thank you for your detailed screenshots with the devtools included. Unfortunately the screenshots doesn't show which rule(with their respective selector + styles) is being applied that does the highlighting. Could you make a new screenshot (with dark reader enabled) with the same element selected and being highlighted by the web instead of by the browser as currently shown in the screenshots (those with devtools included).

Sure, can you please let me know if any of these screenshots are sufficient?

<img width="2050" alt="Screen Shot 2022-09-01 at 8 00 16 AM" src="https://user-images.githubusercontent.com/
<img width="2054" alt="Screen Shot 2022-09-01 at 8 01 14 AM" src="https://user-images.githubusercontent.com/88589076/187909082-3075d999-bfb7-4f11-b18d-7eec84e12bde.png">
88589076/187908886-5be6cb09-8464-4ce3-9deb-750acb2f02fb.png">
<img width="2055" alt="Screen Shot 2022-09-01 at 8 01 31 AM" src="https://user-images.githubusercontent.com/88589076/187909075-512daf32-1ee3-49f7-b036-5b382a3e8f73.png">

**Comment by faroutnovelty** [comment]
<img width="2052" alt="Screen Shot 2022-09-01 at 8 04 00 AM" src="https://user-images.githubusercontent.com/88589076/187909491-4e36da8b-da7d-4d8b-955c-ef64528a4020.png">

**Comment by faroutnovelty** [comment]
<img width="2054" alt="Screen Shot 2022-09-01 at 8 04 28 AM" src="https://user-images.githubusercontent.com/88589076/187909583-a705b8e8-2193-4b0d-981d-d17728bed772.png">

**Comment by faroutnovelty** [comment]
Only when I click the blue highlighting and the context menu associated with it pops up does the blue highlighting color show up correctly. When I close the context menu or click something else, the blue highlighting shows up as grey highlighting again.
<img width="2055" alt="Screen Shot 2022-09-01 at 8 09 51 AM" src="https://user-images.githubusercontent.com/88589076/187910556-2407ba59-d0ed-4464-8e50-2c9c7908b3c1.png">

**Comment by faroutnovelty** [comment]
I can also select the highlighting in dev tools and provide screenshots of the entire styles tab if that would be helpful.

**Comment by faroutnovelty** [comment]
> I can also select the highlighting in dev tools and provide screenshots of the entire styles tab if that would be helpful.
Here's my attempt at doing this. I selected the blue highlight in dev tools and then scrolled from the top to the bottom of the styles pane (scrolling down further in each screenshot).

<img width="2056" alt="Screen Shot 2022-09-01 at 9 35 13 AM" src="https://user-images.githubusercontent.com/88589076/187927894-15ed9608-a8a2-4b67-bf20-95f75fc703fb.png">

<img width="2055" alt="Screen Shot 2022-09-01 at 9 35 29 AM" src="https://user-images.githubusercontent.com/88589076/187927921-4e0a98ac-26e6-46dd-8dc4-51e894b33ed4.png">

<img width="2055" alt="Screen Shot 2022-09-01 at 9 35 44 AM" src="https://user-images.githubusercontent.com/88589076/187927944-abede6d3-302c-4786-afe3-dfbc91275362.png">

<img width="2056" alt="Screen Shot 2022-09-01 at 9 36 17 AM" src="https://user-images.githubusercontent.com/88589076/187927951-cd86b4c9-e6e6-4b02-b0d5-baf8488c2da7.png">

<img width="2056" alt="Screen Shot 2022-09-01 at 9 36 36 AM" src="https://user-images.githubusercontent.com/88589076/187927981-98deb7ac-4060-4aac-99c4-bed97cd9a4f2.png">

**Comment by faroutnovelty** [comment]
And this is how the blue highlighting should look (with Dark Reader enabled, it only currently only looks this way when I click it which opens the highlight modification prompt).

<img width="2056" alt="Screen Shot 2022-09-01 at 6 20 12 PM" src="https://user-images.githubusercontent.com/88589076/188022711-fca427f4-8f59-4154-a520-e8a44cb52c5e.png">

**Comment by faroutnovelty** [comment]
It renders correctly with dark reader enabled if I change
.co_hl.blue {
    background-color: rgb(45, 49, 51) !important;
}

to

.co_hl.blue {
    background-color: rgb(19, 71, 115) !important;
}

<img width="2056" alt="Screen Shot 2022-09-01 at 6 22 43 PM" src="https://user-images.githubusercontent.com/88589076/188022983-11a4f7ad-41d3-47df-925c-efb15d312ba6.png">

**Comment by faroutnovelty** [comment]
And here's the fix:

================================

1.next.westlaw.com

CSS
.co_hl.blue {
background-color: rgb(19, 71, 115) !important;
}

================================

**Comment by faroutnovelty** [comment]
@Gusted 

Here's the fix for blue highlighting on Westlaw Edge:

================================

1.next.westlaw.com

CSS
.co_hl.blue {
background-color: rgb(19, 71, 115) !important;
}

================================

**Comment by Gusted** [comment]
Does:
How does it look if you change the fix to:
```
.co_hl.blue {
   background-color: ${lightblue} !important;
}
```

**Comment by faroutnovelty** [comment]
> Does: How does it look if you change the fix to:
> 
> ```
> .co_hl.blue {
>    background-color: ${lightblue} !important;
> }
> ```

See screenshot:
<img width="1176" alt="Screen Shot 2022-09-04 at 8 36 12 AM" src="https://user-images.githubusercontent.com/88589076/188313821-48573766-d9f2-4760-9e94-2eaf701762c5.png">

**Comment by Gusted** [comment]
Great, do you happen to know the other selectors for the other colors? Then I would be able to produce a fix.

**Comment by faroutnovelty** [comment]
Oh I just realized purple highlighting also incorrectly shows as grey. I'll get you those selectors now.

**Comment by faroutnovelty** [comment]
co_hl pink co_selection_0
co_hl yellow co_selection_1
co_hl green co_selection_2
co_hl orange co_selection_3
co_hl purple co_selection_4
co_hl black co_selection_5

<img width="673" alt="Screen Shot 2022-09-04 at 8 44 34 AM" src="https://user-images.githubusercontent.com/88589076/188314124-21adde6b-baa7-4cbe-bb07-c98deaf4c10f.png">
<img width="658" alt="Screen Shot 2022-09-04 at 8 44 48 AM" src="https://user-images.githubusercontent.com/88589076/188314127-a95763ba-fc7d-43e4-be82-324ba606b20f.png">

**Comment by Gusted** [comment]
Could you try:
```
.co_hl.blue {
   background-color: ${lightblue} !important;
}
.co_hl.purple {
   background-color: ${#bd73bd} !important;
}
```

**Comment by faroutnovelty** [comment]
> Could you try:
> 
> ```
> .co_hl.blue {
>    background-color: ${lightblue} !important;
> }
> .co_hl.purple {
>    background-color: ${#bd73bd} !important;
> }
> ```

<img width="678" alt="Screen Shot 2022-09-04 at 8 50 25 AM" src="https://user-images.githubusercontent.com/88589076/188314326-6a21c981-3b49-4cc6-b670-49d038605039.png">

**Comment by Gusted** [comment]
That looks good to me, are you fine with these fixes? Then I would add them to the general config fixes file.

**Comment by faroutnovelty** [comment]
> That looks good to me, are you fine with these fixes? Then I would add them to the general config fixes file.

It's a little unclear to me whether the lighter blue matches the other colors better, but regardless it is far far better than having no fix. Also: the purple selector boxes still seem to be broken on the highlight selector:

<img width="675" alt="Screen Shot 2022-09-04 at 8 53 42 AM" src="https://user-images.githubusercontent.com/88589076/188314657-57e59132-6924-4ab8-80e8-7abcd554e26d.png">

**Comment by faroutnovelty** [comment]
For purple highlight selectors:

highlightBox purpleBox

**Comment by Gusted** [comment]
Could you try:
```css
.co_hl.blue,
.highlightBox.blueBox {
   background-color: ${lightblue} !important;
}
.co_hl.purple,
.highlightBox.purpleBox {
   background-color: ${#bd73bd} !important;
}
```

**Comment by faroutnovelty** [comment]
> ```css
> .co_hl.blue,
> .highlightBox.blueBox {
>    background-color: ${lightblue} !important;
> }
> .co_hl.purple,
> .highlightBox.purpleBox {
>    background-color: ${#bd73bd} !important;
> }
> ```

Looks great!
<img width="703" alt="Screen Shot 2022-09-04 at 9 15 30 AM" src="https://user-images.githubusercontent.com/88589076/188315430-517e7056-428b-4257-b903-fc5f59c61239.png">

**Comment by Gusted** [comment]
Alrighty, fix is over at #9746 many thanks for your collaborative help!

**Comment by faroutnovelty** [comment]
> Alrighty, fix is over at #9746 many thanks for your collaborative help!

Likewise! As promised, the donation has been made. Really appreciate your time on this.
